index.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-var Requested = require('./requested')
-  , https = require('https')
-  , http = require('http');
+module.exports = (function () {
+  if ('XMLHttpRequest' in global) {
+    return require('./browser');
+  }
 
-module.exports = Requested.extend({
-  initialize: function initialize() {
+  var XMLHttpRequest = global.XMLHttpRequest = require('node-http-xhr');
```

Comment:
@3rd-Eden @aspyrx what if we turn `browser.js` into `index.js` and add the detection of which XMLHttpRequest method to use in the Requests.xhr method rather than setting that method here.